pkg/products/cloudresources/reconciler.go

```diff
@@ -212,7 +217,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		}
 	}
 
-	phase, err = r.reconcileCloudResourceStrategies(client)
+	maintenanceDay, _, err := addon.GetStringParameter(ctx, client, operatorNamespace, MaintenanceDay)
+	if err != nil {
+		return phase, err
+	}
+
+	maintenanceHour, _, err := addon.GetStringParameter(ctx, client, operatorNamespace, MaintenanceHour)
+	if err != nil {
+		return phase, err
+	}
```

Comment:
```suggestion
	maintenanceDay, _, err := addon.GetStringParameter(ctx, client, r.ConfigManager.GetOperatorNamespace(), MaintenanceDay)
	if err != nil {
		return phase, err
	}

	maintenanceHour, _, err := addon.GetStringParameter(ctx, client, r.ConfigManager.GetOperatorNamespace(), MaintenanceHour)
	if err != nil {
		return phase, err
	}
```
Think this should be the RHOAM operator namespace where the addon parameter secret would be 🤔

Reply:
This variable has been set at the start of the reconcile function `operatorNamespace := r.Config.GetOperatorNamespace()` so I am using that here.

Reply:
Nope, these are different. `operatorNamespace := r.Config.GetOperatorNamespace()` is the CRO operator namespace ("redhat-rhoam-cloud-resource-operator"). It is the `ConfigManager` rather than the `Config` if we want to use the RHOAM operator namespace ("redhat-rhoam-operator") where the addon parameters secret would be 🤔

Reply:
Ok no problem, I will update it. Thank you for explaining.